src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Conversions/RecoveryPointConversions.cs

```diff
@@ -61,7 +64,7 @@ public class RecoveryPointConversions
                         RecoveryPointId = rp.Name,
```

Comment:
there are few additional field in detail thats why we kept it separate. I will check the possibility and do it in next checkin